tslint.json

```diff
@@ -1,6 +1,9 @@
 {
     "extends": "@ssense/tslint-config",
     "rules": {
-        "no-any": false
+        "no-any": false,
+        "prefer-template": false,
+        "cyclomatic-complexity": false,
```

Comment:
Can't we keep this rule enabled ? Why did you have to disable it ?

Reply:
The functions are basically too big because of many IFs... Break them into minor functions won't change much the complexity.
This lint didin't exist before.

Reply:
cool